Makefile.in

```diff
@@ -513,167 +515,131 @@ endif
 	$(INSTALL) -m 644 fish.pc $(DESTDIR)$(datadir)/pkgconfig
-	for i in $(COMPLETIONS_DIR_FILES:%='%'); do \
-		$(INSTALL) -m 644 $$i $(DESTDIR)$(datadir)/fish/completions/; \
-		true; \
-	done;
-	for i in $(FUNCTIONS_DIR_FILES:%='%'); do \
-		$(INSTALL) -m 644 $$i $(DESTDIR)$(datadir)/fish/functions/; \
-		true; \
-	done;
-	for i in share/man/man1/*.1; do \
-		$(INSTALL) -m 644 $$i $(DESTDIR)$(datadir)/fish/man/man1/; \
-		true; \
-	done;
-	for i in share/tools/*.py; do\
-		$(INSTALL) -m 755 $$i $(DESTDIR)$(datadir)/fish/tools/; \
-		true; \
-	done;
-	for i in share/tools/web_config/*; do\
-		$(INSTALL) -m 644 $$i $(DESTDIR)$(datadir)/fish/tools/web_config/; \
-		true; \
-	done;
-	for i in share/tools/web_config/js/*; do\
-		$(INSTALL) -m 644 $$i $(DESTDIR)$(datadir)/fish/tools/web_config/js/; \
-		true; \
-	done;
-	for i in share/tools/web_config/partials/*; do\
-		$(INSTALL) -m 644 $$i $(DESTDIR)$(datadir)/fish/tools/web_config/partials/; \
-		true; \
-	done;
-	for i in share/tools/web_config/sample_prompts/*.fish; do\
-		$(INSTALL) -m 644 $$i $(DESTDIR)$(datadir)/fish/tools/web_config/sample_prompts/; \
-		true; \
-	done;
-	for i in share/tools/web_config/*.py; do\
-		$(INSTALL) -m 755 $$i $(DESTDIR)$(datadir)/fish/tools/web_config/; \
-		true; \
-	done;
+	$(foreach i, $(COMPLETIONS_DIR_FILES:%='%'), $(INSTALL) -m 644 $(i) $(DESTDIR)$(datadir)/fish/completions/;)
+	$(foreach i, $(FUNCTIONS_DIR_FILES:%='%'), $(INSTALL) -m 644 $(i) $(DESTDIR)$(datadir)/fish/functions/;)
+	$(foreach i, $(wildcard share/man/man1/*.1), $(INSTALL) -m 644 $(i) $(DESTDIR)$(datadir)/fish/man/man1/ || true)
+	$(foreach i, $(wildcard share/tools/*.py), $(INSTALL) -m 755 $(i) $(DESTDIR)$(datadir)/fish/tools/;)
+	$(foreach i, $(wildcard share/tools/web_config/*), $(INSTALL) -m 644 $(i) $(DESTDIR)$(datadir)/fish/tools/web_config/;)
+	$(foreach i, $(wildcard share/tools/web_config/js/*), $(INSTALL) -m 644 $(i) $(DESTDIR)$(datadir)/fish/tools/web_config/js/ || true)
+	$(foreach i, $(wildcard share/tools/web_config/partials/*), $(INSTALL) -m 644 $(i) $(DESTDIR)$(datadir)/fish/tools/web_config/partials/ || true)
+	$(foreach i, $(wildcard share/tools/web_config/sample_prompts/*), $(INSTALL) -m 644 $(i) $(DESTDIR)$(datadir)/fish/tools/web_config/sample_prompts/ || true)
+	$(foreach i, $(wildcard share/tools/web_config/*.py), $(INSTALL) -m 755 $(i) $(DESTDIR)$(datadir)/fish/tools/web_config/ || true)
 
 	$(INSTALL) -m 755 -d $(DESTDIR)$(docdir)
-	for i in user_doc/html/* ChangeLog; do \
-		if test -f $$i; then \
-			$(INSTALL) -m 644 $$i $(DESTDIR)$(docdir); \
-		fi; \
-	done;
+	$(foreach i, $(wildcard user_doc/html/*), $(INSTALL) -m 644 $(i) $(DESTDIR)$(docdir);)
+	$(INSTALL) -m 644 CHANGELOG.md $(DESTDIR)$(docdir)
+	
 	$(INSTALL) -m 755 -d $(DESTDIR)$(mandir)/man1
-	for i in $(MANUALS); do \
-		$(INSTALL) -m 644 $$i $(DESTDIR)$(mandir)/man1/; \
-		true; \
-	done;
+	$(foreach i, $(MANUALS), $(INSTALL) -m 644 $(i) $(DESTDIR)$(mandir)/man1/;)
```

Comment:
I'm guessing this change is the reason Travis is failing. Perhaps revert this portion.